cheroot/test/test_core.py

```diff
@@ -717,6 +729,13 @@ def test_expose_decorator(self):
         self.assertStatus(200)
         self.assertBody('Mr. and Mrs. Watson')
 
+    @classmethod
+    def teardown_class(cls):
+        cls.supervisor.stop()
+        # remove the temp static files
+        for p in [favicon_ico_file, index_html_file]:
```

Comment:
It's not a good idea to create files globally in module outside of the class and remove it in testcase class.